cypress/support/a11y/checkAxe.ts

```diff
@@ -21,20 +38,23 @@ function logViolations(violations: Result[]) {
     } ${violations.length === 1 ? 'was' : 'were'} detected.`
   );
 
-  // Destructure keys from the violations object to create a readable table
-  const violationData = violations.map(({ id, description, impact, nodes }) => ({
-      id,
-      description,
-      impact,
-      nodes: nodes.length
-    })
-  );
-
   // Print the table of violations to the console
   cy.task('table', violationData);
 }
 
-Cypress.Commands.add('checkAxe', (context, axeConfig, callback) => {
+function logViolationsAndThrow(violations: Result[]) {
+  _handleViolations(violations);
+}
+
+function logViolationsToConsoleOnly(violations: Result[]) {
+  _handleViolations(violations, true);
+}
+
+Cypress.Commands.add('checkAxe', (skipFailures, context, axeConfig, callback) => {
```

Comment:
See above comment about object arg
```suggestion
Cypress.Commands.add('checkAxe', ({ skipFailures, context, axeConfig, callback } = {}) => {
```